code/game/objects/items/debugger.dm

```diff
@@ -0,0 +1,59 @@
+//The debugger! Can be used to test pretty much anything. Make a subtype with its own effects under debug().
+//Fun fact: It took 3 pull requests for this to finally get added in.
+
```

Comment:
I dont agree that we need a debugger object tbh.
most debug stuff I do is just shoddily crafted verbs that are deleted when they fulfil their use, you don't need a framework or item for that.